abstract/trace-context-overview.md

```diff
@@ -2,6 +2,6 @@
 
 Trace context represented by set of name/value pairs describing identity of every http request. As a performance optimization measure few of these pairs were promoted to a separate header. This header has fixed length and defined sequence of fields.

```

Comment:
is the file name still correct?

Reply:
it is an overview of both headers. Two of them makes a trace context.